lib/galaxy/tools/search/__init__.py

```diff
@@ -57,15 +57,25 @@
 from whoosh.writing import AsyncWriter
 
 from galaxy.config import GalaxyAppConfiguration
-from galaxy.util import ExecutionTimer
-from galaxy.web.framework.helpers import to_unicode
+from galaxy.util import (
+    ExecutionTimer,
+    unicodify,
+)
 
 log = logging.getLogger(__name__)
 
 CanConvertToFloat = Union[str, int, float]
 CanConvertToInt = Union[str, int, float]
 
 
+def to_unicode(a_string):
```

Comment:
I am pretty sure there is no need for this function because:
- it could be replaced with just `unicodify()` as it misuses the encoding parameter ("utf-8"), which is the source encoding, not the desired encoding.
- all calls to this functions are not needed any way, as it's used on `Tool` attributes which are strings.

Reply:
Thanks! I will remove it then, I just did the minimal move here but it makes sense if it was not that useful in the first place.